data/json/items/comestibles/carnivore.json

```
@@ -1038,6 +1038,91 @@
     "snippet_category": "cooked_mutant_lung_desc",
     "delete": { "flags": [ "RAW" ] }
   },
+  {
+    "id": "heart",
+    "type": "COMESTIBLE",
+    "copy-from": "flesh",
+    "looks_like": "offal",
+    "name": { "str": "piece of raw heart", "str_pl": "pieces of raw heart" },
+    "description": "A portion of an animal's heart.  It's tough but not that chewy, and it spoils very quickly.",
+    "weight": "55 g",
+    "volume": "50 ml",
+    "color": "pink",
+    "spoils_in": "1 day",
+    "price_postapoc": "6 cent",
+    "fun": -10,
+    "calories": 50,
+    "vitamins": [ [ "vitC", 0 ], [ "calcium", 3 ], [ "iron", 14 ], [ "meat_allergen", 1 ] ],
```

Which animal's heart did you base nutrition on?

```suggestion
    "calories": 50,
    "vitamins": [ [ "vitC", "3 mg" ], [ "calcium", "2840 μg" ], [ "iron", "2660 μg" ], [ "meat_allergen", 1 ] ],
```

https://fdc.nal.usda.gov/fdc-app.html#/food-details/168267/nutrients
~Code suggestion based on raw pork heart